x/ccv/provider/types/msg.go

```diff
@@ -883,12 +881,8 @@ func ValidateInitializationParameters(initializationParameters ConsumerInitializ
 	return nil
 }
 
-// validateHash validates a hash
-func validateHash(hash []byte) error {
-	if len(hash) == 0 {
-		return fmt.Errorf("hash cannot be empty")
-	}
-	if len(hash) > MaxHashLength {
+func ValidateByteSlice(hash []byte, maxLength int) error {
+	if len(hash) > maxLength {
```

Comment:
Why don't we care if it's 0 anymore?

Reply:
To allow for the binary hash and the genesis hash to be empty as they are not checked anywhere. If we start doing something with them, then we add them.